src/compose/types/page-block/report.ts

```diff
@@ -0,0 +1,38 @@
+import { PageBlock, PageBlockInput, Registry } from './base'
+import { Apply, CortezaID, NoID } from '../../../cast'
+
+const kind = 'Report'
+
+interface Options {
+  reportID: string;
+  scenarioID: string;
+  blockID: string;
+  elementID: string;
```

Comment:
I would personally persist just the minimum subset of IDs to get it working.
So, `reportID`, `scenarioID` and `elementID`.

I would omit `blockID` since elements already have unique IDs.

This would come in handy if we decide to allow elements to move between blocks.